server/templates/pages/src/style-guide.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="referrer" content="origin">
+        <meta name="robots" content="noindex,nofollow">
+        <ink rel="stylesheet" href="/styles/main.css">
+    </head>
+    <body>
+        <label class="mdl-checkbox mdl-js-checkbox" for="checkbox-1">
+          <input type="checkbox" id="checkbox-1" class="checkbox_test">
+          <span class="checkbox_label">Checkbox 1</span>
+        </label>
+        <label class="mdl-checkbox mdl-js-checkbox" for="checkbox-2">
+          <input type="checkbox" id="checkbox-2" class="checkbox_test">
+          <span class="checkbox_label">Checkbox 2</span>
+        </label>
+        <br/>
+        <label class="mdl-radio mdl-js-radio" for="option-1">
```

Comment:
these `mdl-` classes are only used for the checkbox. Do not use them in your radio input CSS, instead use a wrapper class such as `fxa-radio-group` or you might not even need that class. It might work if you just write the styles for just `input[type="radio"] {....}`